CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ A declarative library for application development using cloud services.
   - Migrate APIPlugin's websocket provider to use AppSyncRealTimeClient [#330](https://github.com/aws-amplify/amplify-ios/pull/330), [#341](https://github.com/aws-amplify/amplify-ios/pull/341), [#352](https://github.com/aws-amplify/amplify-ios/pull/352)
 - **DataStore**
   - DataStore.save() API now takes in a condition [#355](https://github.com/aws-amplify/amplify-ios/pull/355)
```

Comment:
`- DataStore.save() with condition [#355](https://github.com/aws-amplify/amplify-ios/pull/355)`

for consisent wording

Reply:
I'm assuming you wanted me to overwrite the changes you recently submitted? if so.. I think we are now good to go